src/target/riscv/riscv-013.c

```diff
@@ -120,7 +120,8 @@ typedef enum slot {
 typedef struct {
 	struct list_head list;
 	int abs_chain_position;
-
+	/* The base address to access this DM on DMI */
+	uint32_t dbgbase;
```

Comment:
Is this used anywhere? You set it, but then everywhere dbgbase is needed you get it from target instead of dm.

Reply:
use in `get_dm`
https://github.com/riscv/riscv-openocd/blob/be70adca6f20ea20d906027a01eda8b2526f21ef/src/target/riscv/riscv-013.c#L244-L245